pkg/common/utils.go

```diff
@@ -0,0 +1,11 @@
+package common
+
+import (
+	"net/url"
+	"strings"
+)
+
+func TablePathEncode(str string) string {
+	return strings.ReplaceAll(
+		strings.ReplaceAll(url.PathEscape(str), ".", "%2E"), "-", "%2D")
```

Comment:
you can use   here
```golang
return strings.NewReplacer(".", "%2E", "-", "%2D").Replace(url.PathEscape(str))
```

Reply:
> you can use here
> 
> ```go
> return strings.NewReplacer(".", "%2E", "-", "%2D").Replace(url.PathEscape(str))
> ```

We can extract this as function and place it in the common package.

Reply:
resolve this comment